lib/components/atoms/Blockquote/Blockquote.js

```diff
@@ -10,17 +10,15 @@ import styled from 'styled-components';
 import styles from './Blockquote.style';
 import type { Props } from './types';
 
-const Blockquote = ({ className, quoteProp, quoteAuthorProp }: Props): Node => (
+const Blockquote = ({ className, quoteContent, quoteAuthor }: Props): Node => (
   <blockquote className={className}>
-    <p className="quoteContent">{quoteProp}</p>
-    <footer className="quoteAuthor">{quoteAuthorProp}</footer>
+    <p className="quoteContent">{quoteContent}</p>
+    <footer className="quoteAuthor">{quoteAuthor}</footer>
   </blockquote>
 );
 
 Blockquote.defaultProps = {
-  quoteProp: '',
-  author: '',
-  className: '',
+  quoteContent: 'lorem ipsum',
```

Comment:
We had discussed quoteAuthor can be optional and thus its default value would be needed. That we will set as
 - Anonymous

Reply:
fixed